modular_skyrat/modules/credits/end.dm

```diff
@@ -0,0 +1,192 @@
+#define CREDIT_ROLL_SPEED 185
+#define CREDIT_SPAWN_SPEED 20
+#define CREDIT_ANIMATE_HEIGHT (14 * world.icon_size)
+#define CREDIT_EASE_DURATION 22
+GLOBAL_VAR_INIT(end_credits_song, null)
+GLOBAL_VAR_INIT(end_credits_title, null)
+GLOBAL_LIST(end_titles)
+#define JOINTEXT(X) jointext(X, null)
+/datum/controller/subsystem/ticker/declare_completion()
+	for(var/client/C)
+		if(!C.credits)
+			C.RollCredits()
+	for(var/thing in GLOB.clients)
```

Comment:
more descriptive var

Reply:
instead of var/thing, do var/client/selected_client or something along those lines